ccpage166.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+# Make a class called Restaurant.
+# The __init__() method for Restaurant should store two attributes: a restaurant_name and a cuisine_type.
+# Make a method called describe_restaurant() that prints these two pieces of information,
+# and a method called open_restaurant() that prints a message indi- cating that the restaurant is open.
+# Make an instance called restaurant from your class . Print the two attri- butes individually, and then call both methods.
+
+class Restaurant():
```

Comment:
@paulghaddad Should there be a double space after the "class" definition?

Reply:
Yep, that's why that violation occurred above.